tests/NetEscapades.EnumGenerators.IntegrationTests/EnumWithDisplayNameInNamespaceExtensionsTests.cs

```diff
@@ -0,0 +1,60 @@
+using Xunit;
+
+namespace NetEscapades.EnumGenerators.IntegrationTests;
+
+public class EnumWithDisplayNameInNamespaceExtensionsTests : ExtensionTests<EnumWithDisplayNameInNamespace>
+{
+    public static TheoryData<EnumWithDisplayNameInNamespace> ValidEnumValues() => new()
+    {
+        EnumWithDisplayNameInNamespace.First,
+        EnumWithDisplayNameInNamespace.Second,
+        (EnumWithDisplayNameInNamespace)3,
+    };
+
+    public static TheoryData<string> ValuesToParse() => new()
+    {
+        "First",
+        "Second",
+        "first",
+        "SECOND",
+        "3",
+        "267",
+        "-267",
+        "2147483647",
+        "3000000000",
+        "Fourth",
+        "Fifth",
+    };
```

Comment:
Thanks for adding this test! This has made me wonder if we should also allow _parsing_ via the `[Display]` attribute too 🤔 Otherwise the values aren't round-trippable. I'm also wondering if we should make using the attributes round-trippable

Reply:
Actually, I need that function, so I was going to suggest the same.

What do you mean by **"attributes round-trippable"**?

Reply:
Done.